src/main/c/generated/tiledb_wrap.cxx

```diff
@@ -8024,6 +8024,18 @@ SWIGEXPORT jint JNICALL Java_io_tiledb_libtiledb_tiledbJNI_tiledb_1array_1schema
 }
 
 
+SWIGEXPORT jint JNICALL Java_io_tiledb_libtiledb_tiledbJNI_tiledb_1stats_1dump_1stdout(JNIEnv *jenv, jclass jcls) {
```

Comment:
this is not swig generated so it should go in the custom cxx file

Reply:
This is swig generated from swig/tiledb_java_extensions.h